jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/FileDataSecretEnricherTest.java

```diff
@@ -80,11 +80,9 @@ public void shouldMaterializeFileContentFrom_Annotation() throws IOException {
 
         final Map<String, String> data = secret.getData();
         assertThat(data)
-            .containsKey(TEST_APPLICATION_PROPERTIES);
-
-        assertThat(data)
-            .containsEntry(TEST_APPLICATION_PROPERTIES, Base64Util
-                .encodeToString(Files.readAllBytes(Paths.get(TEST_APPLICATION_PROPERTIES_PATH))));
+            .containsKey(TEST_APPLICATION_PROPERTIES)
+                .containsEntry(TEST_APPLICATION_PROPERTIES, Base64Util
```

Comment:
I have to remove `.containsKey(TEST_APPLICATION_PROPERTIES)` assertion, now do I have to add any thing so `containsEntry` can verify `TEST_APPLICATION_PROPERTIES` key is present or this will happen as default by `.containsEntry` ?